examples/python/droid_dataset/README.md

```diff
@@ -0,0 +1,11 @@
+<!--[metadata]
+title = "DROID dataset"
+source = "https://github.com/rerun-io/python-example-droid-dataset"
+tags = ["2D", "3D", "Depth", "Pinhole camera", "Blueprint"]
+thumbnail = "https://static.rerun.io/droid-screenshot-thumbnail.png/87462a48d06a3c2fcc9225ac86df617048c262e6/480w.png"
+thumbnail_dimensions = [480, 480]
+-->
+
+This example visualizes the [DROID dataset](https://droid-dataset.github.io/), which is a diverse robot manipulation dataset that contains 350 hours of interaction and detailed sensor data.
+
+https://vimeo.com/940717540?autoplay=1&loop=1&autopause=0&background=1&muted=1&ratio=1920:1080
```

Comment:
This example README needs some expansion. We've updated how we write these a bit since 0.15. For examples of how we write these now for in-repo examples, see e.g. Controlnet. For external projects (like this one), check out the ROS bridge and VRS examples (you'll find there we still have some room to go on consistency but we're making progress). It should have this structure:

1. Single line explanation
2. Screenshot or video
3. Background section
4. (Optional) "Logging and visualizing with Rerun" section
5. "Run the code" section